### PR TITLE
 VazhovIA_lab1

### PR DESCRIPTION
**Pull request by L0sted**

### CODE REVIEW

File: VazhovIA/lab1/calculator/main.cpp [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        cout << left * right << endl;
+        break;
+    case '/':
+        if (right == 0) {
+            cout << "Division by zero is forbidden" << endl;
+            return 8;
+        }
+        cout << left / right << endl;
+        break;
+    }
```
Comment (Toxa-man) [inline VazhovIA/lab1/calculator/main.cpp]: There is no validation of the entered sign

File: VazhovIA/lab1/calculator/main.cpp [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+int main()
+{
+    float left, right;
+    char sign;
+    cin >> left >> sign >> right;
+
+    switch (sign)
+    {
+    case '+':
+        cout << left + right << endl;
```
Comment (Toxa-man) [inline VazhovIA/lab1/calculator/main.cpp]: Store result of arithmetic operation in separated variable and print before app closing

File: VazhovIA/lab1/calculator/main.cpp [hunk 3]
```diff
@@ -5,28 +5,32 @@ using namespace std;
 
 int main()
 {
-    float left, right;
+    float left, right, result;
     char sign;
     cin >> left >> sign >> right;
```
Comment (Toxa-man) [inline VazhovIA/lab1/calculator/main.cpp]: That is really unclear for user, what program is expecting from him.

File: VazhovIA/lab1/calculator/main.cpp (hunk 1 above)
Comment (Toxa-man) [reply VazhovIA/lab1/calculator/main.cpp]: Not fixed.

Comment (Toxa-man) [reply VazhovIA/lab1/calculator/main.cpp]: User won't be able to see this message as application will shutdwon immediately

File: VazhovIA/lab1/calculator/main.cpp [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        }
+        result = left / right;
+        break;
+    default:
+        cout << "Wrong sign" << endl;
+        return 1;
+        break;
+    }
+    cout << result << endl;
+    #ifdef __linux__ //im using linux so i have no getch or system("pause")
```
Comment (Toxa-man) [inline VazhovIA/lab1/calculator/main.cpp]: You can use this SO solution
https://stackoverflow.com/questions/7469139/what-is-equivalent-to-getch-getche-in-linux

File: VazhovIA/lab1/calculator/main.cpp (hunk 1 above)
Comment (Toxa-man) [reply VazhovIA/lab1/calculator/main.cpp]: Not fixed

File: VazhovIA/lab1/calculator/main.cpp (hunk 4 above)
Comment (L0sted) [reply VazhovIA/lab1/calculator/main.cpp]: this solution is preety hard to understand for me and probably unreal to defense if you'll ask, can i just rewrite this app using ncurses library?

**Pull request merged**